docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx

```diff
@@ -3,5 +3,53 @@ title: "AWS Elastic Container Service"
 sidebarTitle: "AWS ECS"
 ---
 
-You can refer to the instructions found [here](/production/deploying-to-cloud/aws/setup).
-Make sure you use the `aws-dev` Terraform directory.
\ No newline at end of file
+## Overview
+
+The most common AWS deployment of Mage is with AWS ECS. Our Terraform scripts manage the process of creating an ECS cluster, a load balancer, and a database. [Here's a link](/production/deploying-to-cloud/aws/resources) to the default output of `terraform plan` on AWS ECS.
+
+<Frame height="400" caption="An architecture diagram of a Mage AWS ECS deployment.">
+  <img alt="Contributing" src="/media/architecture/aws-ecs.png" />
+</Frame>
+
+
+We use the following AWS services in our deployment:
+
+## Elastic Container Service (ECS)
+
+Amazon ECS is a service provided by AWS for managing and deploying containerized applications. The ECS service in Terraform is used to define and configure the ECS clusters and services. 
+
+ECS allows you to run containers at scale, and the Terraform configuration for ECS service includes specifying the task definition, desired task count, launch type, and other settings related to container orchestration.
+
+The Mage application runs as multiple containers in an Amazon ECS instance. 
```

Comment:
You can say ECS service instead of ECS instance.

Reply:
ECS stands for elastic container service, so I think that's redundant

Reply:
No, that's not redundant. https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html